wgsl/index.bs

```diff
@@ -3959,6 +3959,269 @@ TODO: deduplicate these tables
 
 ## Texture built-in functions ## {#texture-builtin-functions}
 
+### Sample ### {#texture-sample-functions}
+
+```
+<result> textureSample(<texture>, <sampler>, <coordinates> [, <named-arguments>])
```

Comment:
Reminder, for this and other function-like things, we use arrow syntax to declare the return type.

     textureSample(...) -> <result>